modules/doc/content/framework_development/authorship_guidelines.md

```diff
@@ -0,0 +1,74 @@
+in the relevant journal article(s) that should be used by all, both within the community and externally,
+to cite the MOOSE framework and its modules. Please see the [citing.md] page for the current versions
+of these articles. These articles are published and updated on a regular basis.
+
+## MOOSE framework paper authorship guidelines
+
+When citing MOOSE, whether when using it, using a derived application, or just referring to
+the MOOSE project, the top-level MOOSE paper is the article that should be cited. It is featured in the [Citing MOOSE](citing.md#MOOSE) page.
+This paper is regularly replaced with either a new paper or an "update" paper. "Update" papers
+refer to a summary of changes related to a periodic update of the software from the timeframe of the previous "update" paper, and can have a different authorship list as the original
+paper.
+
+In order to best acknowledge the current contributors to the framework, we strive to keep the authorship
+of the MOOSE framework paper representative. To do that, we publish a summary of the new developments in the framework
+and use the list of the principal contributors since the previous paper as the list of authors for the new paper.
+
+The contributors are listed in two categories.
+
+- Semi-permanent contributors are included in every new MOOSE framework paper by default. These contributors
+  have written a very large part of the MOOSE framework and must be credited, at least until their contributions
+  are overtaken by others.
+
+The current criteria for semi-permanent authorship are listed below:
+
+- Top 10 contributor as measured by lines of code in the `framework/` and other utility folders or
+- Top 10 contributor as measured by number of commits in the `framework/` folder
+
+
+!alert note
+While it seems poor practice to recognize contribution by lines of code or commit numbers, this criteria did not
+exist when these contributions were made, so the metric did not influence the coding style of these authors.
+These criteria may be reformulated in the future in terms of contributed features and impact.
+
+
+- Transient contributors are included in the next MOOSE framework paper for the period(s) in which they contributed.
+  These contributors have played an important role in MOOSE development since the previous paper was published.
+
+
+The current guidelines for transient authorship are listed below:
+
+- 1 major contribution or
+- 4 minor contributions and significant involvement in MOOSE-related activities or
+- 10 fixup contributions and significant involvement in MOOSE-related activities
+
+
+Contributions are only tallied for the *framework and supporting utilities*. Modules are deliberately excluded,
+as developers of a module should publish a module paper and report on new developments outside of the MOOSE framework paper.
+MOOSE-related activities are developments in the MOOSE modules and MOOSE applications.
```

Comment:
I think this sentence is confusing on the backs of the other two. It could be less confusing if you edit the bulleted list to say "1 major framework contribution", "4 minor framework contributions", etc.

Reply:
If you do that, then I would delete the first sentence of this paragraph and then probably swap what are currently sentences two and three